specifications/api/swagger.yaml

```diff
@@ -1863,17 +1909,23 @@ paths:
         - *parametersAddressState
       responses: *responsesListAddresses
 
-  /stake-pools:
+  /wallets/{walletId}/stake-pools:
     get:
       operationId: listStakePools
       tags: ["Stake Pools"]
       summary: List
       description: |
         <p align="right">status: <strong>stable</strong></p>
 
-        List all known stake pools ordered by descending `desirability`.
-        Some pools _may_ also have metadata attached to them if they have been registered
-        to the metadata registry.
+        List all known stake pools ordered by descending `non_myopic_member_rewards`.
+
+        Some pools _may_ also have metadata attached to them.
+
+        The `non_myopic_member_rewards` — and thus the ordering — depends on the balance of the given wallet.
```

Comment:
Small note, I'd actually swap these two sentences. So that it we give precision on the ordering after actually mentioning that pools are ordered by descending member rewards.

Reply:
List all known stake pools ordered by descending `non_myopic_member_rewards`. The `non_myopic_member_rewards` — and thus the ordering — depends on the balance of the given wallet.

> /!\ On the incentivized testnet, pools are instead ordered by descending `desirability`.
        
Some pools _may_ also have metadata attached to them.